extras/userdoc/md/documentation/using-nest-with-music.md

```diff
@@ -24,12 +22,11 @@ Jeanette Hellgren Kotaleski, and Örjan Ekeberg. Run-time interoperability
 between neuronal simulators based on the MUSIC framework.
 Neuroinformatics, 8, 2010. doi:10.1007/s12021-010-9064-z* and available from [here](http://www.springerlink.com/content/r6j425027lmv1251/).
 
-Sending and receiving spike events
------------------------------------
+## Sending and receiving spike events
 
 A minimal example for the exchange of spikes between two independent instances
 of NEST is given in the example `examples/nest/music/minimalmusicsetup.music`.
-It uses one [SLI script](../an_introduction_to_sli/index.html "An Introduction
+It uses one [SLI script](an-introduction-to-sli.md "An Introduction
```

Comment:
Sorry, could not be found by my regular expression, because there was a linebreak.